app/helpers/bookmarks_helper.rb

```diff
@@ -13,6 +13,11 @@ def bookmark_if_exists(bookmarkable)
     end
   end  
   
+  def bookmark_exists?(bookmarkable)
```

Comment:
Pull request looks good! The name of this method is my only quibble -- bookmark_exists? makes me think "is there a bookmark for this?" rather than "does the current user have this bookmarked?" I'm not sure what else I would suggest, though. /o\

If you can think of something, that would be great, but if not, let me know and I can mark this ready to merge as it is.